### PR TITLE
[MO] fix UnaryElementwise reverse_infer

### PR DESCRIPTION
**Pull request by pavel-esir**
**Description**: for UnaryElementwise operations by default is used reverse_infer for binary elementwise operation. The lates one tries to access input ports with index 1 which do not exist in unary elementwise ops
**Solution**: call `reverse_bypass_infer` in `UnaryElementwise`, which is designed exactly for this reason to pass exact shape value form single out port into input port.

Ticket: xxx-101680 and issue https://github.com/openvinotoolkit/openvino/issues/15138

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names
* [x]  Transformation preserves tensor names

Validation:
* [x]  Unit tests: done
* [x]  Framework operation tests n/a
* [x]  Transformation tests: no new transformations

Documentation:
* [x]  Supported frameworks operations list: n/a
* [x]  Guide on how to convert the **public** model n/a
* [x]  User guide update: n/a

### CODE REVIEW

**Review by mvafin — APPROVED** [review]
I wonder how `UnaryElementwise` operation class is different from `Activation`.

File: tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py [hunk 1]
```diff
@@ -53,10 +53,12 @@ def test_eltwise_infer(self, value1, shape1, value2, shape2, shape_infer, exp_va
...
-                             'node_1': {'shape': shape_array(value1).shape if value1 is not None else shape_array(shape1),
-                                        'value': value1},
-                             'node_2': {'shape': shape_array(value2).shape if value2 is not None else shape_array(shape2),
-                                        'value': value2}
+                             'node_1': {
+                                 'shape': shape_array(value1).shape if value1 is not None else shape_array(shape1),
+                                 'value': value1},
+                             'node_2': {
+                                 'shape': shape_array(value2).shape if value2 is not None else shape_array(shape2),
+                                 'value': value2}
```
Comment (rkazants) [inline tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py:61]: not needed to change

File: tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 from openvino.tools.mo.front.common.partial_infer.eltwise import eltwise_infer, eltwise_reverse_infer
 from openvino.tools.mo.front.common.partial_infer.utils import shape_array, strict_compare_tensors, \
-    dynamic_dimension_value
+    dynamic_dimension_value, reverse_bypass_infer
```
Comment (rkazants) [inline tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py:11]: please check Python formatting:  not sorted

File: tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py (hunk 1 above)
Comment (pavel-esir) [reply tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py:61]: autoformat changed it, returned back

**Comment by pavel-esir** [comment]
> I wonder how `UnaryElementwise` operation class is different from `Activation`.

indeed, for shape and type inference no difference at all

**Review by rkazants — APPROVED** [review]

File: tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py (hunk 2 above)
Comment (rkazants) [reply tools/mo/unit_tests/mo/front/common/partial_infer/eltwise_test.py:11]: It should be like this:

```
from openvino.tools.mo.front.common.partial_infer.utils import dynamic_dimension_value, reverse_bypass_infer, shape_array, strict_compare_tensors,
```

**Pull request merged**